### PR TITLE
Release v4.3.8

### PR DESCRIPTION
**Pull request by dvaldivia**
Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

**Review by harshavardhana — APPROVED** [review]

**Comment by harshavardhana** [comment]
@dvaldivia some issue with the code changes

```
# github.com/minio/operator/pkg/controller/cluster
pkg/controller/cluster/main-controller.go:221:2: undefined: klog
pkg/controller/cluster/main-controller.go:223:32: undefined: klog
pkg/controller/cluster/main-controller.go:255:2: undefined: klog
pkg/controller/cluster/main-controller.go:364:5: undefined: klog
pkg/controller/cluster/main-controller.go:368:6: undefined: klog
pkg/controller/cluster/main-controller.go:372:5: undefined: klog
pkg/controller/cluster/main-controller.go:376:6: undefined: klog
pkg/controller/cluster/main-controller.go:382:3: undefined: klog
pkg/controller/cluster/main-controller.go:386:3: undefined: klog
pkg/controller/cluster/main-controller.go:1513:131: undefined: madmin
pkg/controller/cluster/main-controller.go:386:3: too many errors
# github.com/minio/operator/pkg/controller/cluster
vet: pkg/controller/cluster/main-controller.go:1513:131: undeclared name: madmin
make: *** [Makefile:46: govet] Error 2
Error: Process completed with exit code 2.```

File: pkg/controller/cluster/main-controller.go [hunk 1]
```diff
@@ -1067,10 +1065,10 @@ func (c *Controller) syncHandler(key string) error {
 		if us.CurrentVersion != us.UpdatedVersion {
 			klog.Infof("Updating '%s' MinIO from: %s, to: %s",
 				tenantName, us.CurrentVersion, us.UpdatedVersion)
-			// In case the upgrade is from an older version to RELEASE.2021-07-27T02-40-15Z (which introduced
+			// In case the upgrade is from an older version to RELEASE.2021-12-09T06-19-41Z-27T02-40-15Z (which introduced
 			// MinIO server integrated with Console), we need to delete the old console deployment and service.
 			// We do this only when MinIO server is successfully updated and
-			unifiedConsoleReleaseTime, _ := miniov2.ReleaseTagToReleaseTime("RELEASE.2021-07-27T02-40-15Z")
+			unifiedConsoleReleaseTime, _ := miniov2.ReleaseTagToReleaseTime("RELEASE.2021-12-09T06-19-41Z")
```
Comment (harshavardhana) [inline pkg/controller/cluster/main-controller.go:1071]: why are we changing this? do we not need this for console pod detection?

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Review by harshavardhana — APPROVED** [review]

File: docs/crd.adoc [hunk 2]
```diff
@@ -4,10 +4,10 @@
 [id="{p}-api-reference"]
 == API Reference
 
-:minio-image: https://hub.docker.com/r/minio/minio/tags[minio/minio:RELEASE.2021-11-09T03-21-45Z]
+:minio-image: https://hub.docker.com/r/minio/minio/tags[minio/minio:RELEASE.2021-12-09T06-19-41Z]
```
Comment (harshavardhana) [inline docs/crd.adoc:7]:
```suggestion
:minio-image: https://hub.docker.com/r/minio/minio/tags[minio/minio:RELEASE.2021-12-18T04-42-33Z]
```

File: helm/operator/values.yaml [hunk 3]
```diff
@@ -34,7 +34,7 @@ operator:
 console:
   image:
     repository: minio/console
-    tag: v0.12.3
+    tag: v0.12.8
```
Comment (harshavardhana) [inline helm/operator/values.yaml:37]:
```suggestion
    tag: v0.12.9
```

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: helm/tenant/templates/tenant.yaml [hunk 4]
```diff
@@ -20,7 +20,7 @@ metadata:
     name: {{ dig "scheduler" "name" "" . }}
   {{ end }}
 spec:
-  image: {{ dig "image" "repository" "minio/minio" . }}:{{ dig "image" "tag" "RELEASE.2021-10-06T23-36-31Z" . }}
+  image: {{ dig "image" "repository" "minio/minio" . }}:{{ dig "image" "tag" "RELEASE.2021-12-18T04-42-33Z" . }}
```
Comment (Alevsk) [inline helm/tenant/templates/tenant.yaml:23]: This is the MinIO version that contains the bug affecting the root account

**Review by Alevsk — CHANGES_REQUESTED** [review]

File: helm/tenant/templates/tenant.yaml (hunk 4 above)
Comment (harshavardhana) [reply helm/tenant/templates/tenant.yaml:23]: Yeah let's wait for another release today @Alevsk

File: docs/templates/asciidoctor/gv_list.tpl [hunk 5]
```diff
@@ -7,10 +7,10 @@
 [id="{p}-api-reference"]
 == API Reference
 
-:minio-image: https://hub.docker.com/r/minio/minio/tags[minio/minio:RELEASE.2021-11-09T03-21-45Z]
+:minio-image: https://hub.docker.com/r/minio/minio/tags[minio/minio:RELEASE.2021-12-18T04-42-33Z]
```
Comment (harshavardhana) [inline docs/templates/asciidoctor/gv_list.tpl:10]:
```suggestion
:minio-image: https://hub.docker.com/r/minio/minio/tags[minio/minio:RELEASE.2021-12-20T22-07-16Z]
```

File: examples/kustomization/base/tenant.yaml [hunk 6]
```diff
@@ -103,7 +103,7 @@ spec:
   ## https://github.com/minio/minio/tree/master/docs/tls/kubernetes#2-create-kubernetes-secret
   # externalClientCertSecret: {}
   ## Registry location and Tag to download MinIO Server image
-  image: minio/minio:RELEASE.2021-11-09T03-21-45Z
+  image: minio/minio:RELEASE.2021-12-18T04-42-33Z
```
Comment (harshavardhana) [inline examples/kustomization/base/tenant.yaml:106]:
```suggestion
  image: quay.io/minio/minio:RELEASE.2021-12-20T22-07-16Z
```

File: helm/tenant/templates/tenant.yaml (hunk 4 above)
Comment (harshavardhana) [inline helm/tenant/templates/tenant.yaml:23]:
```suggestion
  image: {{ dig "image" "repository" "minio/minio" . }}:{{ dig "image" "tag" "RELEASE.2021-12-20T22-07-16Z" . }}
```

File: helm/tenant/values.yaml [hunk 7]
```diff
@@ -5,7 +5,7 @@ tenants:
     ## Registry location and Tag to download MinIO Server image
     image:
       repository: minio/minio
-      tag: RELEASE.2021-11-09T03-21-45Z
+      tag: RELEASE.2021-12-18T04-42-33Z
```
Comment (harshavardhana) [inline helm/tenant/values.yaml:8]:
```suggestion
      tag: RELEASE.2021-12-20T22-07-16Z
```

File: helm/tenant/values.yaml [hunk 8]
```diff
@@ -5,7 +5,7 @@ tenants:
     ## Registry location and Tag to download MinIO Server image
     image:
       repository: minio/minio
```
Comment (harshavardhana) [inline helm/tenant/values.yaml:7]:
```suggestion
      repository: quay.io/minio/minio
```

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Review by harshavardhana — APPROVED** [review]

**Review by Alevsk — APPROVED** [review]

**Pull request merged**